### PR TITLE
chore: clean up some comments

### PR DESCRIPTION
**Pull request by KeranYang**
Signed-off-by: Keran Yang <yangkr920208@gmail.com>

Explain what this PR does.

### CODE REVIEW

File: pkg/sources/generator/tickgen.go [hunk 1]
```diff
@@ -215,13 +215,7 @@ func (mg *memgen) IsEmpty() bool {
 	return len(mg.srcchan) == 0
 }
 
-// Read reads a chunk of messages and returns at the first occurrence of an error. Error does not indicate that the
```
Comment (KeranYang) [inline pkg/sources/generator/tickgen.go:218]: Already clarified in the `Read` interface.

File: pkg/isb/forward/shutdown.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func (isdf *InterStepDataForward) Stop() {
 // ForceStop sets up the force shutdown flag.
 func (isdf *InterStepDataForward) ForceStop() {
 	// call stop (what if we have an enthusiastic shutdown that forces first)
-	// eg. I know I have written a wrong UDF shutdown ASAP
+	// e.g. I know I have written a wrong UDF shutdown ASAP
```
Comment (vigith) [inline pkg/isb/forward/shutdown.go:69]:
```suggestion
	// e.g. I know I have written a wrong UDF, so shutdown ASAP
```

File: pkg/sources/generator/tickgen.go [hunk 3]
```diff
@@ -215,13 +215,7 @@ func (mg *memgen) IsEmpty() bool {
...
-// array of result is empty, the callee should process all the elements in the array even if the error is set. Read
-// will not mark the message in the buffer as "READ" if the read for that index is erring.
-// There is a chance that we have read the message and the container got forcefully terminated before processing. To provide
-// at-least-once semantics for reading, during restart we will have to reprocess all unacknowledged messages.
-//
-// the context passed to read should be different from the lifecycle context that is used by this vertex.
+// The context passed to read should be different from the lifecycle context that is used by this vertex.
```
Comment (vigith) [inline pkg/sources/generator/tickgen.go:218]: golang function comment should start with the function name.

**Review by vigith — APPROVED** [review]

File: pkg/sources/generator/tickgen.go (hunk 3 above)
Comment (KeranYang) [reply pkg/sources/generator/tickgen.go]: +1. Addressed by removing this piece of comment. The context passed to read is not used anyway.

**Pull request merged**